src/main/java/by/stetsko/ApplicationRunner.java

```diff
@@ -0,0 +1,12 @@
+package by.stetsko;
+
+import by.stetsko.config.ApplicationConfig;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+public class ApplicationRunner {
+
+    public static void main(String[] args) {
+        final AnnotationConfigApplicationContext context = new AnnotationConfigApplicationContext(ApplicationConfig.class);
```

Comment:
Here you should retrieve operations and perform them (or inject set of all operations and process them in 1 foreach cycle)